dask_jobqueue/core.py

```diff
@@ -123,7 +123,7 @@ class Job(ProcessInterface, abc.ABC):
 
 %(job_header)s
 %(env_header)s
-%(worker_command)s
+exec %(worker_command)s
```

Comment:
I don't get it, and I have to admit I'm reluctant to add this change.

Reply:
I will try to remove this part from the PR. It is a direct consequence of changing this to actually use the script instead of "interpreting" it: https://github.com/dask/dask-jobqueue/blob/8264b8cd6fa028f35665a554c5fea4be34a97781/dask_jobqueue/local.py#L53-L59 . My guess is that it could make a second PR.